src/en/help-openstack.md

```diff
@@ -20,6 +20,10 @@ juju add-cloud
 ```
 ...will enter the interactive mode. Enter the desired values to continue.
 
+!!! Note:
+To use the authentication from OpenStack, find the URL in your novarc the
```

Comment:
That rewording is an improvement. I've had coffee since writing the original and agree with your assessment.

As I consider a better location for the information, we must remember that they need this information during the interactive `add-cloud`. Putting the note after the interactive bits is slightly late, IMO. Perhaps moving it above the command itself would be better. That is, unless I misunderstand the process and you can autoload-credentials BEFORE you add the cloud to Juju or can use autoload-credentials without adding the cloud at all.